tests/packagedcode/data/podspec/nanopb.podspec.expected.json

```diff
@@ -0,0 +1,45 @@
+{
+  "type": "pods",
+  "namespace": null,
+  "name": "nanopb",
+  "version": "1.30905.0",
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Objective-C",
+  "description": " Nanopb is a small code-size Protocol Buffers implementation Nanopb is a small code-size Protocol Buffers implementation in ansi C. It is especially suitable for use in Nanopb is a small code-size Protocol Buffers implementation Nanopb is a small code-size Protocol Buffers implementation in ansi C. It is especially suitable for use in microcontrollers, but fits any memory restricted system.",
+  "release_date": null,
+  "parties": [
+    {
+      "type": "person",
+      "role": "author",
+      "name": "Petteri Aimonen",
+      "email": "jpa@nanopb.mail.kapsi.fi",
+      "url": null
+    }
+  ],
+  "keywords": [],
+  "homepage_url": "https://github.com/nanopb/nanopb",
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": "https://github.com/nanopb/nanopb.git",
+  "copyright": null,
+  "license_expression": "unknown",
+  "declared_license": ":type => zlib, :file => LICENSE.txt",
+  "notice_text": null,
+  "root_path": null,
+  "dependencies": [],
+  "contains_source_code": null,
+  "source_packages": [
+    "https://github.com/nanopb/nanopb.git"
+  ],
+  "purl": "pkg:pods/nanopb@1.30905.0",
+  "repository_homepage_url": "https://cocoapods.org/pods/nanopb",
+  "repository_download_url": "https://github.com/nanopb/nanopb/archive/1.30905.0.zip",
+  "api_data_url": "https://github.com/nanopb/nanopb"
```

Comment:
Is this instead something from from https://github.com/CocoaPods/Specs ? you should check in the pod Ruby code to understand how things work.